recipes-core/busybox/files/poweroff.cfg

```diff
@@ -0,0 +1,2 @@
+# Used by stubdom
+CONFIG_POWEROFF=y
```

Comment:
Nit: `CONFIG_POWEROFF` is defined under `Init Utilities` in the `defconfig`, so it would be defined in `init.cfg` if we follow OE upstream standard for configuration artefacts.

Reply:
Oh, I didn't know about the convention.  That makes sense.

Hmmm, one wrinkle is oe-core already has `init.cfg` which is conditionally included:
`${@["", "file://init.cfg"][(d.getVar('VIRTUAL-RUNTIME_init_manager') == 'busybox')]}`

I don't think we want to set, `VIRTUAL-RUNTIME_init_manager`.  I guess we could just extended SRC_URI with `init.cfg`.  What do you want?

Reply:
That would conflict if we ended up fiddling with `VIRTUAL-RUNTIME_init_manager`...
I think the easier is to keep `poweroff.cfg` for now until we end up using `VIRTUAL-RUNTIME_init-manager` for real.